plutus-contract/test/Spec/TxConstraints/MustPayToPubKeyAddress.hs

```diff
@@ -1,52 +1,91 @@
-{-# LANGUAGE DataKinds           #-}
-{-# LANGUAGE LambdaCase          #-}
-{-# LANGUAGE NumericUnderscores  #-}
-{-# LANGUAGE OverloadedStrings   #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE TemplateHaskell     #-}
-{-# LANGUAGE TypeApplications    #-}
-{-# LANGUAGE TypeFamilies        #-}
+{-# LANGUAGE AllowAmbiguousTypes    #-}
```

Comment:
Is this a leftover from putting LanguageContext in a type class? Are all these added pragma's are still needed?